src/Features/Core/Portable/NavigateTo/AbstractNavigateToSearchService.cs

```diff
@@ -52,7 +64,7 @@ internal abstract partial class AbstractNavigateToSearchService : INavigateToSea
             }
 
             await SearchDocumentInCurrentProcessAsync(
-                document, searchPattern, kinds, onResultFound, isFullyLoaded, cancellationToken).ConfigureAwait(false);
+                document, searchPattern, kinds, onItemFound, isFullyLoaded, cancellationToken).ConfigureAwait(false);
```

Comment:
both the remote side and the local side only deal with RoslynNavigateToItem internally.  Only on the boundary (this method) do we convert it to the INavigateToResult that consumers need.